test/fixtures/bugfix-175/expected-unsafe-wrap-es6.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+const sharedPropType = process.env.NODE_ENV !== "production" ? PropTypes.number : {};
```

Comment:
I'm surprised we handle this variable.

Reply:
This is part of the work I did in https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types/pull/146

If we don't wrap it, the minifier won't fully remove it because it might have side-effects (e.g. function calls or getters).

![image](https://user-images.githubusercontent.com/195534/52141172-b719e700-2609-11e9-8ac8-18eaf3df860b.png)

Handling this with the wrap behavior allows it to be minified out completely:

![image](https://user-images.githubusercontent.com/195534/52141240-edeffd00-2609-11e9-9a38-a3c53559a132.png)

There might be a way to use the `/*@__PURE__*/` annotation instead to produce the desired result, which I think would clean up a lot of this plugin, but I was unable to get that to work effectively enough in the REPL.